EasyReflectometry/calculators/refl1d/wrapper.py

```diff
@@ -2,7 +2,7 @@
 
 from typing import Tuple
 
-from easyCore import np
+from easyscience import np
```

Comment:
same as above

Reply:
Thanks for pointing this out.  I forgot.

Reply:
Did a search and replace in repo